ci-operator/step-registry/ovn/conf/dualstack/ovn-conf-dualstack-commands.sh

```diff
@@ -20,3 +20,5 @@ networking:
   - 172.30.0.0/16
   - fd00:172:16::/112
 EOF
+
+export OPENSHIFT_INSTALL_EXPERIMENTAL_DUAL_STACK="true"
```

Comment:
It seems like either this or the `env:` clause above must be unnecessary?

Reply:
this is not necessary, I've added only to be able to trigger the job and verify the environment variable was plumbed correctly 
ci/rehearse/openshift/cluster-network-operator/release-4.9/e2e-azure-ovn-dualstack — Job

Reply:
the only way to cascade values seems to define the variable in the job, and set it its value in some of the steps https://docs.ci.openshift.org/docs/architecture/step-registry/#hierarchical-propagation